run-ab-platform.sh

```diff
@@ -61,47 +61,80 @@ readonly eventTypeRefresh="refresh"
 readonly eventTypeUninstall="uninstall"
 
 telemetrySuccess=false
-telemetrySessionULID=""
+telemetrySessionUUID=""
+# Deprecated, use telemetryUserUUID instead
+# only here for backwards compatability reasons, can be removed when the following issues have been resolved:
+# - https://github.com/airbytehq/airbyte-internal-issues/issues/7758
+# - https://github.com/airbytehq/PyAirbyte/issues/219
 telemetryUserULID=""
+telemetryUserUUID=""
```

Comment:
Discussed on Zoom. Renaming this to `analyticsID` (probably spelled `analytics_id` in the Yaml) will resolve my concern about conflation of the two fields, and we can still over time migrate specific applications from one to the other.

```suggestion
telemetryUserULID=""
analyticsID=""
```